server/config/config.go

```diff
@@ -288,6 +288,12 @@ type RemoteExecutionConfig struct {
 	EnableRedisAvailabilityMonitoring bool               `yaml:"enable_redis_availability_monitoring" usage:"If enabled, the execution server will detect if Redis has lost state and will ask Bazel to retry executions."`
 }
 
+type DockerDeviceMapping struct {
+	PathOnHost         string  `yaml:"path_on_host" usage:"path to device that should be mapped from the host."`
+	PathInContainer    string  `yaml:"path_in_container" usage:"path under which the device will be present in container."`
+	CgroupPermissions  string  `yaml:"cgroup_permissions" usage:"cgroup permissions that should be assinged to device."`
```

Comment:
s/assinged/assigned/g

Reply:
Thank you for catching this - typos are bane of my existence! Fixed.